module/bookCollection.js

```diff
@@ -0,0 +1,25 @@
+class BookCollection {
+  constructor() {
+    this.collectionData = JSON.parse(window.localStorage.getItem('collectionArray')) || [];
+    this.removeButtons = [];
+  }
+
+  addBook(book) {
+    this.collectionData.push(book);
+    this.updateStorage();
+  }
```

Comment:
- [ ] You have done well to ensure that the books are added to the collection when added 👏 . However, I suggest ensuring that when the book is added, the input fields are cleared. This will make the user experience easy and the project will look more professional ☺️ .